doc/man1/common/submit-other-options.rst

```diff
@@ -34,6 +34,18 @@ OTHER OPTIONS
    character, then VAL is interpreted as a file, which must be valid JSON,
    to use as the attribute value.
 
+**--add-file=[NAME=]ARG**
+   Add a file to the RFC 37 file archive in jobspec before submission. If no
+   ``NAME`` is provided, then ``ARG`` is assumed to be the path to a local
```

Comment:
For the case where file content comes from a file, it might be helpful emphasize (somehow) that a snapshot of the file is taken at job submission time and that after that changing or removing the file has no effect on the job.

Reply:
How is this?
>    Add a file to the RFC 37 file archive in jobspec before submission. Both
>   the file metatdata and content are stored in the archive, so modification
>   or deletion of a file after being processed by this option will have no
 >  effect on the job.

Reply:
Perfect.